Source/ARTCrypto.h

```diff
@@ -28,13 +28,20 @@ NS_ASSUME_NONNULL_BEGIN
 @end
 
 /**
- ARTCipherParams contains configuration options for a channel cipher, including algorithm, mode, key length and key.
- Ably client libraries currently support AES with CBC, PKCS#7 with a default key length of 256 bits. All implementations also support AES128.
+ The ARTCipherParams object contains properties to configure encryption for a channel.
```

Comment:
```suggestion
 The `ARTCipherParams` object contains properties to configure encryption for a channel.
```

@tbedford @m-hulbert I’d suggest that this sort of formatting should be in the canonical documentation, what do you think?

Reply:
Agree with the formatting. It is already styled as code in the canonical docs though.

Reply:
This will link to the page that user is already on. Is this what was expected? @lawrence-forooghian